microshift_cli_ref/microshift-usage-oc-kubectl.adoc

```diff
@@ -6,36 +6,36 @@ include::_attributes/attributes-microshift.adoc[]
+You can use the `kubectl` CLI tool to interact with Kubernetes primitives on your {microshift-short} cluster. You can also use existing `kubectl` workflows and scripts for users coming from another Kubernetes environment, or for those who prefer to use the `kubectl` CLI.
 
-The `kubectl` CLI tool is included in the archive if you download the `oc` CLI tool.
+* The `kubectl` CLI tool is included in the archive when you download `oc`.
 
-For more information, read the link:https://kubernetes.io/docs/reference/kubectl/overview/[Kubernetes CLI tool documentation].
+* For more information, see the link:https://kubernetes.io/docs/reference/kubectl/overview/[Kubernetes CLI tool documentation].
 
 [id="microshift-oc-binary_{context}"]
 == The oc CLI tool
 
-The `oc` CLI tool offers the same capabilities as the `kubectl` CLI tool, but it extends to natively support additional {product-title} features, including:
+The `oc` CLI tool offers the same capabilities as the `kubectl` CLI tool, but it extends to natively support additional {OCP} features, including:
 
 * **Route resource**
 +
-The `Route` resource object is specific to {product-title} distributions, and builds upon standard Kubernetes primitives.
+The `Route` resource object is specific to {OCP} distributions, and builds upon standard Kubernetes primitives.
 +
 * **Additional commands**
 +
 The additional command `oc new-app`, for example, makes it easier to get new applications started using existing source code or pre-built images.
 
 [IMPORTANT]
 ====
-If you installed an earlier version of the `oc` CLI tool, you cannot use it to complete all of the commands in {product-title} {ocp-version}. If you want the latest features, you must download and install the latest version of the `oc` CLI tool corresponding to your {product-title} version.
+If you installed an earlier version of `oc`, you might not be able use it to complete all of the commands in {microshift-short} {product-version}. If you want the latest features, you must download and install the latest version of `oc` that corresponds with your {microshift-short} version.
 ====
 
-Non-security API changes will involve, at minimum, two minor releases (4.1 to 4.2 to 4.3, for example) to allow older `oc` binaries to update. Using new capabilities might require newer `oc` binaries. A 4.3 server might have additional capabilities that a 4.2 `oc` binary cannot use and a 4.3 `oc` binary might have additional capabilities that are unsupported by a 4.2 server.
+Non-security API changes involve at least two minor releases, such as from 4.15 to 4.17, for example. But using new capabilities often requires the latest `oc` binary. A 4.17 server might have additional capabilities that a 4.12 `oc` binary cannot use and a 4.17 `oc` binary might have additional capabilities that are unsupported by a 4.13 server.
```

Comment:
Is the word `But` needed in the second sentence ?

Reply:
Yeah....The way it was originally written, I think they meant to imply that `oc` can be used across two releases and still work with non-security API changes, BUT to ensure compatibility with everything else, you should just always update `oc` anyway. TBH that first sentence might not even be true anymore, so I just removed it.